src/cloudimagedirectory/connection/connection.py

```diff
@@ -2,24 +2,23 @@
 import json
 import os
 import pathlib
+from dataclasses import dataclass
 from pathlib import Path
-from typing import Any
 
 exception_not_connected = Exception("Client is not connected to s3 bucket")
 exception_already_connected = Exception("Client is already connected to s3 bucket")
 exception_path_not_existing = Exception("The given path to the filesystem doesn't exist")
 exception_not_implemented = Exception("Not implemented")
 
 
+@dataclass
 class DataEntry:
     """Handles a file from the bucket."""
 
-    # TODO: Set Any to a specific type.
-    def __init__(self, filename: str, content: Any) -> None:
-        """Constructor for DataEntry class."""
-        self.filename = filename
-        self.content = content
+    filename: str
+    content: str
```

Comment:
How does this work? Isn't this field used in other places as a dictionary? :D

Reply:
@miyunari so in a dataclass, you don't need an `__init__` for setting class arguments. You just specify them at the top of the class definition and it takes them in if they're provided.

Reply:
Oh, I meant `content` was a dictionary before, so of type `any` and now you changed it to `string`. Doesn't that cause trouble in other places, where I used `content` as a dict before? :)

Reply:
So the issue was that `test_get_content` was re-using `content` which was typed as a string. That confused `mypy`. I fixed it and switched the `content` argument to be a dictionary or `None`.